example.json

```diff
@@ -57,6 +52,7 @@
                     ]
                 },
                 "_transcript": {
+                    "_comment._inlineTranscript": "Enable item transcript with overrides",
```

Comment:
Not an issue for this component as it's not AAT supported but just FYI the use of '.' is invalid json (for reference I raised the issue on List https://github.com/cgkineo/adapt-list/issues/23).

Perhaps for consistency it's worth using the generic key suggested in the issue?

"_comment": "inlineTranscript = Enable item transcript with overrides",